x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations.test.tsx

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mockKibanaValues, setMockActions, setMockValues } from '../../../../__mocks__';
+import '../../../__mocks__/engine_logic.mock';
+
+import React from 'react';
+
+import { shallow, mount, ReactWrapper } from 'enzyme';
+
+import { EuiBasicTable, EuiEmptyPrompt } from '@elastic/eui';
+
+import { Loading } from '../../../../shared/loading';
+
+import { Curations, CurationsTable } from './curations';
+
+describe('Curations', () => {
+  const { navigateToUrl } = mockKibanaValues;
+
+  const values = {
+    dataLoading: false,
+    curations: [
+      {
+        id: 'cur-id-1',
+        last_updated: 'January 1, 1970 at 12:00PM',
+        queries: ['hiking'],
+      },
+      {
+        id: 'cur-id-2',
+        last_updated: 'January 2, 1970 at 12:00PM',
+        queries: ['mountains', 'valleys'],
+      },
+    ],
+    meta: {
+      page: {
+        current: 1,
+        size: 10,
+        total_results: 2,
+      },
+    },
+  };
+
+  const actions = {
+    loadCurations: jest.fn(),
+    deleteCurationSet: jest.fn(),
+    onPaginate: jest.fn(),
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
Is throwing `jest.clearAllMocks();` in `beforeEach()` preferable to a separate `afterEach`?

Reply:
We do it like this way in nearly all of our tests. I don't know that it's preferable to `afterEach`, but probably good for consistency unless there is a reason to change.